sdk/storage/azfile/file/models.go

```diff
@@ -337,22 +348,31 @@ func (o *StartCopyFromURLOptions) format() (*generated.FileClientStartCopyOption
 			nil, nil
```

Comment:
When o is nil, it is returning NFS headers `FileModeCopyMode` and `FileOwnerCopyMode`, line 343-348. Check if this is needed in case NFS is not used.
```
return &generated.FileClientStartCopyOptions{
  FileModeCopyMode:  to.Ptr(ModeCopyModeSource),
  FileOwnerCopyMode: to.Ptr(OwnerCopyModeSource),
}
```